docs/management/connectors/action-types/opsgenie.asciidoc

```diff
@@ -0,0 +1,173 @@
+
+Opsgenie connectors have the following configuration properties.
+
+Name::      The name of the connector. The name is used to identify a  connector in the management UI connector listing, or in the connector list when configuring an action.
+URL::     The Opsgenie URL: https://api.opsgenie.com. For the EU instance of Opsgenie use: https://api.eu.opsgenie.com. If you are using the <<action-settings, `xpack.actions.allowedHosts`>> setting, make sure the hostname is added to the allowed hosts.
+API Key::   The Opsgenie API authentication key for HTTP Basic authentication. Refer to https://support.atlassian.com/opsgenie/docs/create-a-default-api-integration/[How to generate an Opsgenie API key] for more information.
+
+[float]
+[[opgenie-connector-networking-configuration]]
+==== Connector networking configuration
+
+Use the <<action-settings, Action configuration settings>> to customize connector networking configurations, such as proxies, certificates, or TLS settings. You can set configurations that apply to all your connectors or use `xpack.actions.customHostSettings` to set per-host configurations.
+
+[float]
+[[Preconfigured-opsgenie-configuration]]
+==== Preconfigured connector type
+
+[source,text]
+--
+ my-opsgenie:
+   name: preconfigured-opsgenie-connector-type
+   actionTypeId: .opsgenie
+   config:
+    apiUrl: https://api.opsgenie.com
+   secrets:
+     apiKey: apikey
+--
+
+Config defines information for the connector type.
+
+`apiUrl`:: A string that corresponds to *URL*.
+
+Secrets defines sensitive information for the connector type.
+
+`apiKey`:: A string that corresponds to *API Key*.
+
+[float]
+[[define-opsgenie-ui]]
+==== Define connector in Stack Management
+
+Define Opsgenie connector properties.
+
+[role="screenshot"]
+image::management/connectors/images/opsgenie-connector.png[Opsgenie connector]
+
+Test Opsgenie action parameters.
+
+[role="screenshot"]
+image::management/connectors/images/opsgenie-params-test.png[Opsgenie params test]
+
+[float]
+[[opsgenie-action-configuration]]
+==== Action configuration
+
+The Opsgenie connector supports two types of actions: Create Alert and Close Alert. The properties supported for each action are different because Opsgenie defines different properties for each operation.
+
+When testing the Opsgenie connector, choose the appropriate action from the selector. Each action has different properties that can be configured.
+
+Action::    Whether to create or close an alert within Opsgenie. When using the Opsgenie connector for a rule action, the Action selector is hidden and automatically set to the appropriate value. If the action is run when a rule fires, it will create an alert within Opsgenie. If the action is run when a rule recovers, it will close an alert within Opsgenie.
+
+[float]
+[[opsgenie-action-create-alert-configuration]]
+===== Configure the create alert action
+
+You can configure the create alert action through the form view or using a JSON editor.
```

Comment:
Though I know this page is following the example of the example connector pages, IMO rather than listing each field and meaning here, it's more important to (1) have the appropriate API documentation up-to-date with these properties and descriptions, and (2) have tooltips in the UI for any fields that aren't obvious (e.g. "alias", "entity", "source"). If you want me to add those tooltips in this PR, @jonathan-buttner let me know. Otherwise, I'll create a follow-up PR.

Reply:
Great point @lcawl . Let's do a follow up PR to address that. I'll add these fields in the docs here and we can remove them once we have the tooltips.